Makefile

```
@@ -51,7 +53,121 @@ type: ## Run mypy type checker
+
+.PHONY: docker-build docker-push docker-clean docker-prune docker-compose-up docker-compose-down docker-compose-logs docker-compose-ps docker-compose-up-prod docker-compose-stop
+
+docker-build: ## Build backend Docker image
+	@docker build -t $(APP_NAME)-api:latest -f Dockerfile .
+	@echo "✓ Backend image: $(APP_NAME)-api:latest"
+
+docker-build-frontend: ## Build frontend Docker image
+	@docker build -t $(APP_NAME)-web:latest -f Dockerfile.frontend .
+	@echo "✓ Frontend image: $(APP_NAME)-web:latest"
+
+docker-build-all: docker-build docker-build-frontend ## Build all images
```

checkmake (0.3.2)

[warning] Target "docker-build-all" should be declared PHONY.

(phonydeclared)